recipe/meta.yaml

```diff
@@ -15,30 +15,37 @@ source:
   {{ hash_type }}: {{ hash_val }}
 
 build:
-  noarch: python
-  number: 1
+  number: 0
   script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [py<37]
 
 requirements:
-  build:
+  host:
     - python
     - pip
+    - setuptools
+    - wheel
 
   run:
     - python
     - flask
-    - bcrypt
+    - bcrypt >=3.1.1
 
 test:
   imports:
     - flask_bcrypt
+  requires:
+    - pip
+  commands:
+    - pip check
 
 about:
-  home: https://github.com/maxcountryman/flask-bcrypt
+  home: https://flask-bcrypt.readthedocs.io/en/1.0.1/
+  license: BSD-3-Clause
   license_file: LICENSE
   license_family: BSD
-  license: BSD 3-Clause
   summary: 'Bcrypt hashing for Flask.'
+  description: 'Flask-Bcrypt is a Flask extension that provides bcrypt hashing utilities for your application.'
```

Comment:
The single quote symbol `'` is not needed for the string. It's redundant

Reply:
Okay, thanks! I was following the format of the summary, but I will remove the single quotes from both the summary and description.